lib/universal-search.js

```diff
@@ -70,6 +74,24 @@ Search.prototype = {
     }
   },
 
+  _enableSearchSuggestions: function(win) {
+    // Enable the browser.search.suggest.enabled pref. Save the previous state
+    // so we can restore it on unload.
+    const searchSuggestIsEnabled = Services.prefs.getBoolPref('browser.search.suggest.enabled');
```

Comment:
Good catch; I didn't realize there was a new pref used for UnifiedComplete. [browser.search.suggest.enabled](http://kb.mozillazine.org/Browser.search.suggest.enabled) has been around for ages but [doesn't appear to be used for anything](https://dxr.mozilla.org/mozilla-central/search?q=browser.search.suggest.enabled&redirect=false&case=false) anymore.